docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |find a person by tags |locate people easily
+
+|`* * *` |user |sort people by name |locate people easily
+
+|`* * *` |user |set a list of favourite people|
+
+|`* * *` |user |see a list of frequently accessed contacts|
+
+|`* * *` |user |get confirmation before I edit or delete contacts|
+
+|`* * *` |user |undo my actions |fix my mistakes
+
+|`* * *` |user |add multiple numbers to a contact |separate their mobile and home numbers
+
+|`* * *` |user |password protect my addressBook |protect my contacts' information
```

Comment:
Perhaps this user story could be `* *` instead? Since this is not critical for the functionality of the addressbook.